File: C Programming/I TECH CLASS/Pointer Function/min_max.c
#include <stdio.h>
void maxmin(int a[]);
void maxmin(int a[])
{
    int lowest,max,i;
    max=lowest=a[0];
    for(i=0;i<5;i++)
    {
        if(lowest>a[i])
        {
            lowest=a[i];
        }
    }
    printf("Minumum Number is %d\n",lowest);

    for(i=0;i<5;i++)
    {
        if(max<a[i])
        {
            max=a[i];
        }
    }
    printf("Maximum Number is %d\n",max);
}

int main() {
    int a[5],i;
    printf("Enter The 5 Numbers\n");
    for(i=0;i<5;i++)
    {
        scanf("%d",&a[i]);
    }
    maxmin(a);
}

File: Java/Array/minmax.java
package Java.Array;

import java.util.Scanner;

public class minmax {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a [] = new int[5];
        int min,max;
        System.out.println("Enter 5 Elements of the Array 1");
        for(int i =0;i<5;i++)
        {
            a[i] = sc.nextInt();
        }
        min=max=a[0];
        for(int i =0;i<5;i++)
        {
            if(max<a[i])
            {
               max = a[i];
            }
            if(min>a[i])
            {
                min = a[i];
            }
        }
        System.out.println(max+" is the Maximum Number");
        System.out.println(min+" is the Minimum Number");
        sc.close();
    }
}
